vm/src/builtins/builtinfunc.rs

```diff
@@ -140,9 +143,12 @@ impl PyBuiltinFunction {
     }
 }
 
+// PyCMethodObject in
+// https://github.com/python/cpython/blob/main/Include/cpython/methodobject.h
 #[pyclass(module = false, name = "method_descriptor")]
 pub struct PyBuiltinMethod {
     value: PyNativeFuncDef,
+    mmtype: PyTypeRef,
```

Comment:
I suggest to use just `class` because this field is saving the class owning the method. we don't use those prefixes in RustPython.

Reply:
👍